experimental/cache_line_analysis.cc

```diff
@@ -0,0 +1,95 @@
+#include <stdlib.h>
+#include <time.h>
+
+#include <algorithm>
+#include <iostream>
+#include <memory>
+#include <random>
+#include <string>
+
+// TODO: give compilation instructions instead of including nonlocal files
```

Comment:
(BTW definitely worth fixing that next week :D)

Reply:
depending on the destination this might change 
for now I leave it as TODO